docs/run/nodes.md

```diff
@@ -1,48 +1,45 @@
+# Hosting a node
+This guide describes how to spin up and run a hosted Ceramic node in JavaScript that can be used as a remote node by the [JS HTTP Client](../clients/javascript/http.md) or the [CLI](../clients/javascript/cli.md). It can also be used for redundant stream pinning and replication from any other node.
 
+## **Installation**
+This guide requires the use of a console, [Node.js](https://nodejs.org/en/){:target="_blank"} v14, and [npm](https://www.npmjs.com/get-npm){:target="_blank"} v6. Make sure to have these installed on your machine.
 
-## **Prerequisites**
-
-This installation guide will use a terminal, [Node.js](https://nodejs.org/en/){:target="_blank"} v14, and [npm](https://www.npmjs.com/get-npm){:target="_blank"} v6. Make sure to have these installed on your machine.
+While npm v7 is not officially supported, you may still be able to get it to work. You will need to install the `node-pre-gyp` package globally. This is required until `node-webrtc` which [IPFS](../learn/glossary.md#ipfs) depends on [is upgraded](https://github.com/node-webrtc/node-webrtc/pull/694){:target="_blank"}.
 
-While npm v7 is not officially supported, you may be able to get it to work anyway, however you will need to have the `node-pre-gyp` package installed globally:
 ```bash
 npm install -g node-pre-gyp
 ```
-This is required until node-webrtc (which IPFS depends on) [is upgraded](https://github.com/node-webrtc/node-webrtc/pull/694).
-
-
-## **Installation**
 
+### 1. Install the CLI
 Open your console and install the CLI.
 
 ``` bash
 $ npm install -g @ceramicnetwork/cli
 ```
 
-
-## **Start the daemon**
-
-Next, start the Ceramic daemon.
+### 2. Start the Ceramic node
+This starts a Ceramic node with a standard configuration on the [Clay Testnet](../learn/networks.md#clay-testnet) at `https://localhost:7007`. Behind the scenes a properly configured [IPFS](../learn/glossary.md#ipfs) instance is started, then IPFS is used to create an instance of the Ceramic [JS Core Client](../clients/javascript) connected to the [Clay Testnet](../learn/networks.md#clay-testnet). Once Ceramic Core is running, an HTTP server is started that serves the Ceramic [HTTP API](../reference/http-api.md).
 
 ```bash
 $ ceramic daemon
 ```
 
-This command starts a Ceramic node with a standard cofiguration. By default the daemon runs a bundled IPFS instance. First, a properly configured IPFS instance is started. Then, IPFS is used to create an instance of Ceramic core. Once Ceramic core is running, an HTTP server is started that serves the Ceramic HTTP API. The node is now ready to accept requests from the local environment or remotely.
-
-### Using an external IPFS node
-
-It's also possible to run a separate service for IPFS using the `@ceramicnetwork/ipfs-daemon` npm package. 
+### 3. Use an external IPFS node

```

Comment:
Feels like this should be a sub-section of "Start the Ceramic node" above rather than its own top-level section given that it's an optional step

Reply:
I played with that, but there are so many things that are optional that I thought it was worth them having their own sections, and just calling out right in the beginning that it's optional. I'm going to hold on this change for now, but I know Val/Joel have stories to work on documentation for others running nodes so it can be improved when that happens this sprint or next.

Reply:
Also the documentation here of just using the package isn't enough. We should provide more explicit instructions. To be done when this page is worked on by others